pytorch_lightning/core/saving.py

```diff
@@ -86,11 +86,11 @@ def load_from_checkpoint(
                 These will be converted into a :class:`~dict` and passed into your
                 :class:`LightningModule` for use.
 
-                If your model's `hparams` argument is :class:`~argparse.Namespace`
+                If your model's ``hparams`` argument is :class:`~argparse.Namespace`
                 and .yaml file has hierarchical structure, you need to refactor your model to treat
-                `hparams` as :class:`~dict`.
+                ``hparams`` as :class:`~dict`.
             strict: Whether to strictly enforce that the keys in :attr:`checkpoint_path` match the keys
-                returned by this module's state dict. Default: `True`.
+                returned by this module's state dict. Default: ``True``.
```

Comment:
Sphinx already includes the defaults

```suggestion
                returned by this module's state dict.
```